rxlifecycle-kotlin/src/main/java/com/trello/rxlifecycle/kotlin/rxlifecycle.kt

```diff
@@ -31,19 +31,19 @@ fun <T> Observable<T>.bindToLifecycle(view: View): Observable<T>
         = this.compose<T>(RxLifecycleAndroid.bindView(view))
 
 fun <E> Completable.bindToLifecycle(provider: LifecycleProvider<E>): Completable
-        = this.compose(provider.bindToLifecycle<Completable>().forCompletable())
+        = this.compose(provider.bindToLifecycle<Completable>())
 
 fun <E> Completable.bindUntilEvent(provider: LifecycleProvider<E>, event: E): Completable
-        = this.compose(provider.bindUntilEvent<Completable>(event).forCompletable())
+        = this.compose(provider.bindUntilEvent<Completable>(event))
 
 fun Completable.bindToLifecycle(view: View): Completable
-        = this.compose(RxLifecycleAndroid.bindView<Completable>(view).forCompletable())
+        = this.compose(RxLifecycleAndroid.bindView<Completable>(view))
 
 fun <T, E> Single<T>.bindToLifecycle(provider: LifecycleProvider<E>): Single<T>
-        = this.compose(provider.bindToLifecycle<T>().forSingle<T>())
+        = this.compose(provider.bindToLifecycle<T>())
 
 fun <T, E> Single<T>.bindUntilEvent(provider: LifecycleProvider<E>, event: E): Single<T>
-        = this.compose(provider.bindUntilEvent<T>(event).forSingle<T>())
+        = this.compose(provider.bindUntilEvent<T>(event))
 
 fun <T> Single<T>.bindToLifecycle(view: View): Single<T>
-        = this.compose(RxLifecycleAndroid.bindView<T>(view).forSingle<T>())
\ No newline at end of file
+        = this.compose(RxLifecycleAndroid.bindView<T>(view))
```

Comment:
Missing extension methods for Maybe + Flowable. So far I also didn't see that support was added for those. This is just meant as a reminder.